cabot/cabotapp/tests/tests_basic.py

```diff
@@ -171,6 +184,19 @@ def fake_http_404_response(*args, **kwargs):
     return resp
 
 
+def fake_tcp_success(*args, **kwargs):
+    resp = Mock()
+    resp.query.return_value = Mock()
+    return resp
+
+
+def fake_tcp_failure(*args, **kwargs):
+    resp = Mock()
+    resp.query.return_value = Mock()
+    raise socket.timeout
```

Comment:
I made the change, but not entirely sure I did it the correct way. Slightly confused on how mock is used in this case.